Frontend/ui-driver/src/Screens/HomeScreen/Controller.purs

```diff
@@ -969,6 +969,7 @@ eval (RideActionModalAction (RideActionModal.CancelRide)) state = do
 eval (RideActionModalAction (RideActionModal.CallCustomer)) state = do
   let exophoneNumber = if (take 1 state.data.activeRide.exoPhone) == "0" then state.data.activeRide.exoPhone else "0" <> state.data.activeRide.exoPhone
   updateWithCmdAndExit state [ do
+    -- void $ performTwilioVoiceCall "<rideId>" "<accessToken>" :: TODO :: Once backend APIs are deployed make the changes.
```

Comment:
how will we be handling `accessToken` confidentiality?

Reply:
access token will be generated from backend and it is going to be short lived. For each ride a new accessToken will be generated. 
each access token here is associated with a specific user.